packages/react/src/utils/versions.ts

```diff
@@ -1,9 +1,9 @@
 export const nxVersion = '*';
-export const frameworkVersion = '16.8.3';
-export const typesVersion = '16.8.4';
-export const styledComponentVersion = '4.2.0';
-export const styledComponentTypesVersion = '4.1.15';
-export const emotionVersion = '10.0.10';
-export const domTypesVersion = '16.8.2';
-export const reactRouterVersion = '5.0.0';
-export const testingLibraryVersion = '6.0.0';
+export const frameworkVersion = '16.8.6';
```

Comment:
Please also update our `package.json` with these versions to make sure we're testing against the same versions.